code/__DEFINES/gravity.dm

```diff
@@ -47,7 +47,7 @@
  * This should only be possible on multi-z maps because it works like shit on maps that aren't.
  */
 #define NEGATIVE_GRAVITY -1
-
+// #define NONE 0 Used to indicate no gravity
```

Comment:
Just make a new define `NO_GRAVITY`, I don't like using `NONE` for non-bitlfags

Reply:
https://github.com/tgstation/tgstation/pull/88176/commits/57393265eb84630e9f611c3094ee6c900d1ed767